### DISCUSSION TITLE
Does this compile to Linux ? (.NET Core)

### DISCUSSION THREAD

**Discussion by birojnayak** [otterkit/otterkit-cobol#20] [General Discussion]
Hi @KTSnowy /Gabriel,
Personally, having worked in both COBOL and as a.NET developer, I find this an interesting project.

Before diving further, wanted to understand few things,
1)I believe the objective of this project is to generate a C# code and use msbuild to compile the code to library ? Meaning code is platform agnostic ? (Rectify me)
2)I was also looking at NetCOBOL ? what's the difference according to you ?
3)Do you have sort of roadmap or where you want to take this project ?

Thank you !

**Comment by KTSnowy** [comment]
Hey @birojnayak, sorry for the late reply.

1) Yes, Otterkit will be platform agnostic, it will work on Windows, Linux and macOS (x64 architecture).

2) I don't have much experience with NetCOBOL, but from what I can tell from their website NetCOBOL on Linux is "Standards compliant (COBOL 85, 89 addendum)". Otterkit will support COBOL 2023 ([ISO/IEC 1989:2023](https://www.iso.org/standard/74527.html)), the latest standard. We want to move COBOL forward, so we won't be focusing on legacy standards like COBOL 85.

3) We don't have a formal or detailed roadmap yet, but the goal of this project is to eventually become a production-ready Standard COBOL compiler, one that is free and open source so that the whole ecosystem can benefit from it. We want to bring COBOL back into relevance and modern use, but for that we need to focus on open source and modern standards, we need to focus on Standard COBOL and its latest standard.

Let me know if you any more questions about the project.
